apps/wakucanary/wakucanary.nim

```diff
@@ -24,6 +26,7 @@ const ProtocolsTable = {
 }.toTable
 
 const WebSocketPortOffset = 1000
+const CertsDirectory = "./apps/wakucanary/certs"
```

Comment:
We should make this configurable - what happens if you are not in the nwaku repo root? It will probably fail to write the key and cert, right?